packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/UpdateSliceZoneModal.tsx

```diff
@@ -41,19 +37,16 @@ const UpdateSliceZoneModal: React.FC<UpdateSliceModalProps> = ({
         sliceKeys: slicesInSliceZone.map((slice) => slice.model.id),
       }}
       content={{
-        title: "Update Slice Zone",
+        title: "Update Slices",
       }}
-      omitFooter={!projectHasAvailableSlices}
       dataCy="update-slices-modal"
     >
-      {projectHasAvailableSlices
-        ? ({ values }) => (
-            <UpdateSliceZoneModalList
-              values={values}
-              availableSlices={availableSlices}
-            />
-          )
-        : () => <UpdateSliceZoneModalEmptyState />}
+      {({ values }) => (
```

Comment:
You removed `projectHasAvailableSlices` because it's already handled by the BlankSlate?

Reply:
It's not possible to open this modal if you don't have slice in your project anymore. The button is not even visible :)